@xen-orchestra/lite/src/components/ui/UiCardComingSoon.vue

```diff
@@ -1,8 +1,8 @@
 <template>
-  <UiCard class="pool-dashboard-coming-soon">
+  <UiCard class="ui-card-coming-soon">
     <UiCardTitle>{{ title }}</UiCardTitle>
     <div class="content">
-      <img alt="Under construction" src="@/assets/under-construction.svg" />
+      <img alt="" src="@/assets/under-construction.svg" />
```

Comment:
Because it is redundant with the "Coming soon" text just below and provide no additional useful information.

Reply:
Because it's a required HTML attribute, but it is not required to be filled ^^

Here is a decision tree which I found useful to decide whether we need to fill it: https://www.hmc.edu/communications/accessibility-resources/alt-decision-tree-alternative-text-guide/